Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -336,7 +336,9 @@ def bump_formula_pr
         user_message = ARGV.value("message")
         if user_message
           pr_message += <<~EOS
+
```

Comment:
`brew style` doesn't like the spacing, here.

Reply:
Not really sure what's wrong here; I tried a couple changes but Rubocop always complains...

Reply:
Apparently rubocop doesn't understand that heredocs can start with blank lines, or at least it pretends not to understand that if you use `<<~EOS`.

Reply:
Seems related: https://github.com/bbatsov/rubocop/issues/4041.

Reply:
Either insert the `"\n"` programmatically or add a `# rubocop:disable`